lib/libvmod_std/vmod_std_conversions.c

```diff
@@ -44,6 +44,80 @@
 #include "vtim.h"
 #include "vcc_if.h"
 
+VCL_BYTES v_matchproto_(td_std_bytes)
+vmod_bytes(VRT_CTX, VCL_STRING b, VCL_BYTES d)
+{
+	uintmax_t ret;
+
+	CHECK_OBJ_NOTNULL(ctx, VRT_CTX_MAGIC);
+
+	if (VNUM_2bytes(b, &ret, 0)) {
+		return (d);
+	}
+
+	return (ret);
+}
+
+VCL_STRING v_matchproto_(td_std_bytes2string)
+vmod_bytes2string(VRT_CTX, VCL_BYTES b, VCL_ENUM unit)
+{
+	int i;
+	char *ret;
+	double value;
+	size_t len;
+
+	CHECK_OBJ_NOTNULL(ctx, VRT_CTX_MAGIC);
+
+	if (!strcmp(unit, "B")) {
```

Comment:
the enum comparison could be more efficient by comparing against the enum pointers

Reply:
If we're really going after efficiency, it could be a switch on the first char in unit -- they happen to be all distinct.

Reply:
Clearly the best option updated in new commit, thanks.